jax_tqdm/__init__.py

```diff
@@ -1 +1 @@
-from jax_tqdm.pbar import progress_bar_scan
+from jax_tqdm.pbar import progress_bar_scan, progress_bar_fori_loop
```

Comment:
Do we want to shorten these function names a bit? Though can leave for another PR.